lib/dradis/plugins/content_service/core.rb

```diff
@@ -3,7 +3,7 @@ module Core
     extend ActiveSupport::Concern
 
     included do
-      attr_accessor :logger, :plugin, :project
+      attr_accessor :default_issue_state, :logger, :plugin, :project
```

Comment:
Why is this a `Core` attribute?

How is it different from `default_author` or `default_issue_category`?

Reply:
The difference here is that the `default_issue_state` is a user-defined value that sets the state of all uploaded issues. The `default_author` and `default_issue_category` doesn't seem to be user-defined.

The alternative here is to keep the `default_issue_state` at the Importer level and not pass it to the content service. The downside is, we'd have to make a change in every `content_service.create_issue` call in every upload plugin to pass the `default_issue_state`.